src/use/use.py

```diff
@@ -526,9 +496,7 @@ def _venv_is_win() -> bool:
 
 def _pebkac_no_version_hash(func=None, *, name: str, **kwargs) -> Union[ModuleType,Exception]:
     if func:
-        result = func(name=name, **kwargs)
-        if isinstance(result, ModuleType):
-            return result
+        return func(name=name, **kwargs)
```

Comment:
This doesn't take into account that func could return an Exception, which should NOT be returned in this case.

Reply:
It doesn't happen, though

Reply:
Addressed